### PR TITLE
add tidb_gogc_tuner_max_value and tidb_gogc_tuner_min_value

### PR DESCRIPTION
**Pull request by hawkingrei**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/15375
- Other reference link(s): https://github.com/pingcap/tidb/pull/47148

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -2068,6 +2068,28 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
 - Default value: `ON`
 - This variable controls whether to enable GOGC Tuner.
 
+### tidb_gogc_tuner_max_value <span class="version-mark">New in v7.5.0</span>
```
Comment (Oreoxmt) [inline system-variables.md:2071]: To maintain consistency, please ensure all system variables are listed in alphabetical order. Specifically, position `tidb_gogc_tuner_max_value` and `tidb_gogc_tuner_min_value` ahead of `tidb_gogc_tuner_threshold`.

File: system-variables.md [hunk 2]
```diff
@@ -2068,6 +2068,28 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
...
+
+- Scope: GLOBAL
+- Persists to cluster: Yes
+- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): No
+- Type: Integer
+- Default value: `500`
+- Range: `[10, 2147483647]`
+- The variable is used to control the maximum value of GOGC that the GOGC Tuner can adjust.
+
```
Comment (Oreoxmt) [inline system-variables.md:2080]:
```suggestion
```

File: system-variables.md [hunk 3]
```diff
@@ -2068,6 +2068,28 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
...
+### tidb_gogc_tuner_min_value <span class="version-mark">New in v7.5.0</span>
+
+- Scope: GLOBAL
+- Persists to cluster: Yes
+- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): No
+- Type: Integer
+- Default value: `100`
+- Range: `[10, 2147483647]`
+- The variable is used to control the minimum value of GOGC that the GOGC Tuner can adjust.
+
```
Comment (Oreoxmt) [inline system-variables.md:2091]:
```suggestion
```

**Review by Oreoxmt — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

**Pull request merged**